src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+public class Task {
+    private boolean isDone;
+    private final String str;
+
+    public Task(String str) {
+        this.str = str;
+        this.isDone = false;
+    }
+
+    /**
+     * This marks the task as done
+     * @return String form of task
+     */
+    public String mark() {
+        this.isDone = true;
+        return this.toString();
+    }
+
+    /**
+     * This marks the task as undone
+     * @return String form of task
+     */
+    public String unmark() {
+        this.isDone = false;
+        return this.toString();
+    }
+
+    /**
+     * Custom toString includes checkbox showing whether task is done
+     * @return custom toString
+     */
```

Comment:
Do note that a line break should be added between the method summary and return field.